osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/testutil/TestUtil.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2019 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.testutil;
+
+import static junit.framework.TestCase.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+import org.openmuc.jdlms.AttributeAddress;
+import org.openmuc.jdlms.SelectiveAccessDescription;
+import org.openmuc.jdlms.datatypes.CosemDateFormat;
+import org.openmuc.jdlms.datatypes.DataObject;
+
+public class TestUtil {
```

Comment:
Refactor as custom ArgumentMatcher and rename for example to 'AttributeAddressMatcher'

Reply:
Renamed class name.
Changing to ArgumentMatcher did not result in clearer code, so that's not changed.